### PR TITLE
feat(billing): billing detail of an organization

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

This feature add the crud for the billing detail of an organization.

https://user-images.githubusercontent.com/6163954/222495275-fd1b22fa-05f8-4a93-a0d5-b9703545bf95.mp4

- [ ] testing
- [ ] states list
- [ ] fix the console error bug when we initialise the values of the form

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(billing): billing detail of an organization - 2023-03-02T16:47:55Z](https://new.console.qovery.com/organization/141c07c8-0dd9-4623-983b-3fdd61867255/project/977bd069-20f7-4b24-bd95-28b44174cf43/environment/1eb55c08-8b70-43a5-bba3-a779910c7c54/services/general)

 Another comment will be posted when deployments are terminated

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://z1eb55c08-zc943e245-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://z1eb55c08-z408efb2d-gtw.za8ad0659.bool.sh)

File: libs/pages/settings/src/lib/ui/page-organization-billing/billing-details/billing-details.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { getByDisplayValue, getByTestId } from '@testing-library/react'
+import { render } from '__tests__/utils/setup-jest'
+import { BillingInfo } from 'qovery-typescript-axios'
+import { wrapWithReactHookForm } from '../../../../../../../../__tests__/utils/wrap-with-react-hook-form'
```
Comment (RemiBonnet) [inline libs/pages/settings/src/lib/ui/page-organization-billing/billing-details/billing-details.spec.tsx:4]: ```__tests__/utils/wrap-with-react-hook-form```

File: libs/domains/organization/src/lib/slices/organization/organization.slice.ts [hunk 2]
```diff
@@ -232,6 +233,7 @@ export const organizationSlice = createSlice({
   },
   extraReducers: (builder) => {
     credentialsExtraReducers(builder)
+    billingInfoExtraReducers(builder)
```
Comment (RemiBonnet) [inline libs/domains/organization/src/lib/slices/organization/organization.slice.ts:236]: 👍

File: libs/pages/settings/src/lib/feature/page-organization-billing-feature/billing-details-feature/billing-details-feature.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import { getByLabelText, getByTestId, waitFor } from '@testing-library/react'
+import { render } from '__tests__/utils/setup-jest'
+import { BillingInfo } from 'qovery-typescript-axios'
+import * as storeOrganization from '@qovery/domains/organization'
+import { organizationFactoryMock } from '@qovery/shared/factories'
+import { act, fireEvent } from '../../../../../../../../__tests__/utils/setup-jest'
```
Comment (RemiBonnet) [inline libs/pages/settings/src/lib/feature/page-organization-billing-feature/billing-details-feature/billing-details-feature.spec.tsx:6]: ```__tests__/utils/setup-jest```

**Review by RemiBonnet — APPROVED** [review]
Well done, just one thing can you add a button "save" like other page ? otherwise it's great !

**Pull request merged**